ignite/services/chain/serve.go

```diff
@@ -569,7 +611,12 @@ type CannotBuildAppError struct {
 }
 
 func (e *CannotBuildAppError) Error() string {
-	return fmt.Sprintf("cannot build app:\n\n\t%s", e.Err)
+	// TODO: Find at which point the error is wrapped twice
```

Comment:
Isn't this what `errors.As` is supposed to handle ?

Reply:
Changeset 24cf113

Reply:
I added the comment and condition because at some point the error is wrapped twice. The issue is an old one and I didn't find where it is happening. I think it can be done as a separate issue, maybe as "good first issue" :)